masterbase/lib.py

```diff
@@ -35,9 +38,33 @@ def make_db_uri(is_async: bool = False) -> str:
     return f"{prefix}://{user}:{password}@{host}:{port}/demos"
 
 
-def make_demo_path(session_id: str) -> str:
-    """Make the demo path for the current session."""
-    return os.path.join(DEMOS_PATH, f"{session_id}.dem")
+class DemoSessionManager(BaseModel):
+    """Helper class to facilitate access."""
```

Comment:
The functions and fields inside this class are pretty simple, so don't need good comments for the most part. But the comment for hte class itself could be more descriptive , especially since you implement the `__init__()` method and get to provide documentation there.

Reply:
I had a hunch pydantic was being used incorrectly here anyways so thats the push I need to make this fall back to just a dataclass or likely just a normal class -- Im not validating against user data so makes more sense to just use a normal class tbh